release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -885,6 +904,24 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-builds-bug-fixes"]
 ==== Builds
 
+* Before this update, if you created a build configuration containing an image change trigger in {product-title} 4.7.x or earlier, the image change trigger might trigger builds continuously.
++
+This issue happened because, with the deprecation and removal of the `lastTriggeredImageID` field from the `BuildConfig` spec for Builds, the image change trigger controller stopped checking that field before instantiating builds. {product-title} 4.8 introduced new fields in the status that the image change trigger controller needed to check, but didn't.
++
+With this update, the image change trigger controller continuously checks the correct fields in the spec and status for the last triggered image ID and only triggers a build when necessary. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2004203[BZ#2004203])
```

Comment:
```suggestion
With this update, the image change trigger controller continuously checks the correct fields in the spec and status for the last triggered image ID and thus only triggers a build when necessary. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2004203[BZ#2004203])
```
I find the last part of this sentence to be a bit awkwardly worded. I've added "thus" to emphasize the logical result of the fix, but I'm not sure how well it works. What do you think? Maybe the sentence needs to be split into two?

Reply:
Updated. Thanks.